File: Latihan 3/Python/Processor.py

# Saya Silmi Aulia Rohmah mengerjakan Latihan 3 DPBO 2022 C1 
# dalam mata kuliah DPBO untuk keberkahanNya maka saya tidak melakukan 
# kecurangan seperti yang telah dispesifikasikan. Aamiin 

class Processor:

    def __init__(self): #Constructor

        # atribut
        self.price = 0
        self.name = ""


    # metode getter dan setter

    def getPrice(self):
        return self.price
    
    def setPrice(self, price):
        self.price = price
    
    def getName(self):
        return self.name
    
    def setName(self, name):
        self.name = name
    

    # untuk print atribut kelas Processor
    def PrintProcessor(self):
        print("===== PROCESSOR =====")
        print(" Name  : " + self.getName())
        print(" Price : " + str(self.getPrice()))
    

File: Latihan 3/C++/Processor.cpp
/* Saya Silmi Aulia Rohmah mengerjakan Latihan 3 DPBO 2022 C1 
dalam mata kuliah DPBO untuk keberkahanNya maka saya tidak melakukan 
kecurangan seperti yang telah dispesifikasikan. Aamiin */

#include <iostream>
#include <string>

using namespace std;

class Processor{

    private:
     
        string name;
        int price;

    public:

        Processor(){ //konstruktor kosong

        }

        //metode getter dan setter
        string getName(){

            return this->name;
        }

        void setName(string name){

            this->name = name;
        }

        int getPrice(){

            return this->price;
        }

        void setPrice(int price) {
            this->price = price;

        }

        void PrintProcessor(){ //untuk menampilkan atribut
            cout << "===== PROCESSOR =====" << endl;
            cout << " Name  : " << getName() << endl;
            cout << " Price : " << getPrice() << endl;
        }

        ~Processor(){ //destruktor

        }
};